src/cluster/setup.rst

```diff
@@ -164,9 +164,15 @@ procedure on other machines. Be sure to specify the total number of nodes you
 expect to add to the cluster before adding nodes.
 
 Before you can add nodes to form a cluster, you have to have them
-listen on a public IP address and set up an admin user. Do this, once
+listen on a node address if all nodes can communicate over those IP's,
+also set up an admin user which should be same on all nodes. Do this, once
 per node:
```

Comment:
This is some very awkward wording, IMO. I suggest rewriting the paragraph as:

> Before you can add nodes to form a cluster, you must have them listening on an IP address accessible from the other nodes in the cluster.  Do this once per node:

> In file ``etc/vm.args`` change the the line ``-name couchdb@127.0.0.1`` to ``-name couchdb@<this-nodes-ip-address>`` for each node. For clustered setup, each node in the system must have a unique name.

Reply:
Rephrased but added parameter <FQDN> along with IP @flimzy